lua/nvim-tree.lua

```diff
@@ -27,7 +27,16 @@ function M.focus()
   view.focus()
 end
 
+local function mysub(mystr)
+  if vim.fn.has('win32') == 1 and vim.opt.shellslash._value == true then
+    return mystr:gsub('/', package.config:sub(1, 1))
```

Comment:
What is the result of `:lua print(package.config:sub(1, 1))` when you have shellslash set and not set?

I don't have a windows environment so cannot test.

I'm concerned that we may need to make changes anywhere `path_separator` is used instead of just cwd.